cli/install/install.go

```diff
@@ -434,20 +471,42 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 		}
 
 		if !versionFound {
-			return fmt.Errorf("%s version of tt doesn't exist", ttVersion)
+			commits, err_hashes := getTTCommits(installCtx.Local, distfiles)
+
+			if len(ttVersion) < 7 {
+				return fmt.Errorf("the hash must include at least 7 characters")
+			}
+
+			if err_hashes != nil {
+				return err_hashes
+			}
+
+			for _, com := range commits {
```

Comment:
Don't use abbreviations like `com`. It is unclear what com means.

Reply:
fixed